projects/packages/videopress/src/client/admin/components/edit-video-details/index.tsx

```diff
@@ -33,24 +37,43 @@ const Header = ( { saveDisabled = true }: { saveDisabled?: boolean } ) => {
 	);
 };
 
-const Infos = () => {
+const Infos = ( { video } ) => {
+	const [ title, setTitle ] = useState( video?.title );
+	const [ description, setDescription ] = useState( video?.description );
+	const [ caption, setCaption ] = useState( video?.caption );
+
+	useEffect( () => {
+		setTitle( video?.title );
+		setDescription( video?.description );
+		setCaption( video?.caption );
+	}, [ video ] );
```

Comment:
I think we should avoid using an internal state considering the data is already stored in the state.

Reply:
@retrofox I store in internal state, because user will have the possibility to change it, so the current data starts at initial data.

This useEffect is needed because of the cases that the user come from Media Library, in this case, it starts empty until we fetch the data and then we fill the initial data.